src/components/TextArea/TextArea.md

```diff
@@ -0,0 +1,42 @@
+### Examples
+
+Basic
+```js { "props": { "data-example": "basic" } }
+<TextArea
+  value="This multiline has fixed height so scrollbars may appear."
```

Comment:
can we add lines of content so the scrollbars show in the visual diff (or at least the overflow if Chrome auto-hides scrollbars)?

Reply:
good thinking